test/parallel/test-runner-tap-parser-stream.js

```diff
@@ -4,6 +4,7 @@ const common = require('../common');
 const assert = require('node:assert');
 const { TapParser } = require('internal/test_runner/tap_parser');
 const { TapChecker } = require('internal/test_runner/tap_checker');
+const { tapEscape } = require('internal/test_runner/tap_stream');
```

Comment:
So, I don't think this file is the best place for testing this. This file is more about the TAP parser.

I would test this by doing the following:
- Undo the changes in this file.
- Remove the `tapEscape` export that this PR adds to `lib/internal/test_runner/tap_stream.js`.
- Add a test to `test/message/test_runner_output.js` that uses `\n`, `\r`, etc (`#` and `\` are already tested IIRC).
- Update `test/message/test_runner_output.out` to reflect the changes in the test output.

Reply:
updating this 👍🏽 🚀